test-apps/display-test-app/vite.config.ts

```diff
@@ -32,10 +44,29 @@ Object.keys(packageJson.dependencies).forEach((pkgName) => {
         // filter out pkgs that actually dont have assets
         assets.push(assetsPath);
       }
-    } catch {}
+    } catch { }
+  }
+  if (pkgName.startsWith("@itwin") && !itwinPackagesWithoutAliases.includes(pkgName)) {
+    if (pkgName.includes("electron") || pkgName.includes("mobile")) {
+      const fileName = pkgName.replace("@itwin/", "");
+      packageAliases = { ...packageAliases, [`${pkgName}/lib/cjs/${fileName.split("-")[1].charAt(0).toUpperCase()}${fileName.split("-")[1].substring(1)}Frontend`]: findPathToPackage(pkgName) }
```

Comment:
personally not a fan of the rhs. 

wonder if we should just hardcode these values and add a comment

```
"@itwin/core-electron/lib/cjs/ElectronFrontend": "@itwin/core-electron/src/ElectronFrontend.ts",
"@itwin/core-mobile/lib/cjs/MobileFrontend": "@itwin/core-mobile/src/MobileFrontend.ts",
```